### DISCUSSION TITLE
`LineClamp` with inline expander button

### DISCUSSION THREAD

**Discussion by loxy** [taiga-family/taiga-ui#4036] [Q&A] [answered]
Hi!

There is an example on docs with shows a button to expand the `LineClamp`  under iself. Is it somehow possible to place an expanding button inside the `LineClamp` or even better some text?

Best regards,
Kersten

**Comment by waterplea** [comment, marked as answer]
No — we rely on the CSS line-clamp and it's not possible with it.

**Reply by loxy** [reply]
Ok. Thanks!
